development-docker-compose.yml

```diff
@@ -8,9 +8,9 @@ services:
     ports:
       - 5439:5432
     environment:
-      - POSTGRES_PASSWORD
-      - DATABASE_OWNER
-      - TEAM_PASSWORD
+      - POSTGRES_PASSWORD=${DEVELOPMENT_POSTGRES_PASSWORD}
+      - DATABASE_OWNER=${DEVELOPMENT_DATABASE_OWNER}
+      - TEAM_PASSWORD=${DEVELOPMENT_TEAM_PASSWORD}
```

Comment:
We got rid of prefixing the POSTGRES variables with DEVELOPMENT_ and PRODUCTION_, and pass them in to the travis environment, AWS settings, and the .env (for local testing) in the prefix-less manner. So you will want to remove the DEVELOPMENT_ prefixes and keep them the way they are in the backend-exemplar so it will run correctly locally and in production.